### PR TITLE
'Related content' footer section of 'What's on' page (FE)

### PR DESCRIPTION
**Pull request by janehughes**
## About these changes

Adds relevant templates and cards for the bottom 'related content' section of 'Whats on' page

Design reference:
image

https://www.figma.com/file/pcIxOpwvJ1YRYbt9lbeKUG/TNA-Etna---Design-system?node-id=5786%3A40667&mode=dev

## How to check these changes

- Create 'What's on' page locally
- Preview page locally
- Please note: as the BE is not yet finished for this page, the FE for the cards is hardcoded for testing. The images etc will need adding in the BE to test fully but we are using existing templates and styles for the cards so this should work with real data.

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer
- [ ] Checked PR title starts with ticket number as per project conventions to help us keep track of changes
- [ ] Ensured that PR includes only commits relevant to the ticket
- [ ] Waited for all CI jobs to pass before requesting a review
- [ ] Added/updated tests and documentation where relevant

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

### CODE REVIEW

**Comment by LeanneEmans** [comment]
Hi @janehughes is this PR ready for review please?

**Comment by janehughes** [comment]
> Hi @janehughes is this PR ready for review please?

Not quite yet - I'll aim to finish it off and get it ready for review this week!

File: templates/whatson/whats_on_page.html [hunk 1]
```diff
@@ -87,4 +87,77 @@ <h2 class="tna-heading tna-heading--l event-card-listing__heading">All events</h
       </ul>
     </div>
   </div>
+
+  <div class="section section--pink">
+    <div class="tna-container">
+        <div class="tna-column tna-column--full tna-!--margin-top-l">
+            <h2 class="tna-heading related-content__heading">For professionals</h2>
```
Comment (ahosgood) [inline templates/whatson/whats_on_page.html:94]: I think we might also need a `tna-heading--m`, `tna-heading--l` or `tna-heading--xl` class

Comment (janehughes) [reply templates/whatson/whats_on_page.html]: Hey @ahosgood I've updated with those headings now :) are you happy to sign this off so we can get it merged in, ready for the BE to be updated?

Comment (ahosgood) [reply templates/whatson/whats_on_page.html]: Sorry, of course 👍🏼

**Review by ahosgood — APPROVED** [review]

**Pull request merged**